src/test/kotlin/TestCreateNumalgo2.kt

```diff
@@ -0,0 +1,624 @@
+import org.dif.model.EncodingType
+import org.dif.model.PublicKeyAgreement
+import org.dif.model.PublicKeyAuthentication
+import org.dif.model.PublicKeyTypeAgreement
+import org.dif.model.PublicKeyTypeAuthentication
+import org.dif.peerdid.createPeerDIDNumalgo2
+import org.dif.peerdid.isPeerDID
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class TestCreateNumalgo2 {
+    @Test
+    fun testCreateNumalgo2Positive() {
+        val encryptionKeys = listOf(
+            PublicKeyAgreement(
+                encodedValue = "JhNWeSVLMYccCk7iopQW4guaSJTojqpMEELgSLhKwRr",
+                type = PublicKeyTypeAgreement.X25519, encodingType = EncodingType.BASE58
+            )
+        )
+        val signingKeys = listOf(
+            PublicKeyAuthentication(
+                encodedValue = "ByHnpUCFb1vAfh9CFZ8ZkmUZguURW8nSw889hy6rD8L7",
+                type = PublicKeyTypeAuthentication.ED25519, encodingType = EncodingType.BASE58
+            ),
+            PublicKeyAuthentication(
+                encodedValue = "3M5RCDjPTWPkKSN3sxUmmMqHbmRPegYP1tjcKyrDbt9J",
+                type = PublicKeyTypeAuthentication.ED25519, encodingType = EncodingType.BASE58
+            )
+        )
+        val service = """[
```

Comment:
Please add a positive test where `service` is 
1) non-array JSON object ({ ...})
2) array of one element